JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
+        public getAuthId() {
+            return this.authId;
+        }
+
+        /**
+         * Sets the autheticated user id and the account id in this session.
+         * @param id {string} - The autheticated user id
+         * @param accountId {string} - The account id.
+         * @returns {} 
+         */
+        public setAuthUserContext(id: string, accountId?: string) {
```

Comment:
We debated this in the design, and in the end decided that having a single setting method is better.
- Having 2 sets and 2 clears complicates the API as users will wonder when and how to use each.
- It complicates things as it introduces potentially buggy scenarios where one is set or cleared before/without the other. This way auth context is always set or cleared in one stroke.
- The account id param is clearly documented as an optional parameter in the code. And we will of course show this in the quick-start guide.

All that being said, I cant say how common accountId is going to be..